src/bundle/Resources/public/scss/ui/modules/universal-discovery/_main.scss

```diff
@@ -1,6 +1,6 @@
 .m-ud {
     position: fixed;
-    z-index: 99999;
+    z-index: 1100;
```

Comment:
yeah, from my short investigation looks like UDW will be under: form builder, image editor, PB config panel, PB block action menu.
Risky change.